src/components/PortfolioCard/PortfolioCardDesktop.tsx

```diff
@@ -70,6 +70,8 @@ export const PortfolioCardDesktop = ({
   const classes = useStyles();
   const [isHovered, setHovered] = useState(false);
 
+  const projectImage = transformImage(image, { width: 400, height: 224 });
```

Comment:
Why previously you initialized values `width` and `height` as constants, and here you don't do it like so?

Reply:
There was no need for that as those values are used only once here. In other cases, there were at least two places where I used sizing values.

Reply:
Makes sense, I would probably declare for consistency with other components but I agree that its not necessary